crates/pairing/benches/bls12_381/ec.rs

```diff
@@ -1,8 +1,8 @@
 mod g1 {
     use rand::{Rand, SeedableRng, XorShiftRng};
 
-    use pairing::bls12_381::*;
-    use pairing::CurveProjective;
+    use pairing_ce::bls12_381::*;
```

Comment:
Of course! The crate is named `pairing_ce`. Benches were unmodified since the time it was forked, so they never compiled :) Now they compile. I didn't try to run them though.